packages/react-components/src/components/InputBoxButton.tsx

```diff
@@ -19,6 +19,7 @@ export type InputBoxButtonProps = {
   id?: string;
   ariaLabel?: string;
   tooltipContent?: string;
+  testId?: string;
```

Comment:
Did we modify tests for this component as well?

Reply:
It looks like we don't have any specific tests for `InputBoxButtons`

Reply:
Was just curious why we added a testId for this component?